### PR TITLE
chore: enrich scaling log

### PR DESCRIPTION
**Pull request by Kavinjsir**
### Description

This pull request refactors the log output for successful slot scaling operations to include the following fields:
- `project_name`
- `updated_slots`
- `prev_slots`
- `org_name`

### Motivation

This update addresses the needs outlined in [Issue #386](https://github.com/rilldata/rill-private-issues/issues/386).

### CODE REVIEW

File: admin/worker/run_autoscaler.go [hunk 1]
```diff
@@ -72,7 +72,20 @@ func (w *Worker) runAutoscaler(ctx context.Context) error {
...
+			zap.Int("updated_slots", updatedProject.ProdSlots),
+			zap.Int("prev_slots", targetProject.ProdSlots),
+		}
+
+		updatedProjectOrg, err := w.admin.DB.FindOrganization(ctx, updatedProject.OrganizationID)
+		if err != nil {
+			w.logger.Error("failed to find org for the project", zap.String("project_name", updatedProject.Name), zap.String("org_id", updatedProject.OrganizationID), zap.Error(err))
+		} else {
+			logInfo = append(logInfo, zap.String("org_name", updatedProjectOrg.Name))
+		}
```
Comment (begelundmuller) [inline admin/worker/run_autoscaler.go:86]: Move finding of the org up to after the call to `FindProject`, before calling `UpdateProject`. That way you can just call `continue` if there's an `err`, and the code will be cleaner.

File: admin/worker/run_autoscaler.go [hunk 2]
```diff
@@ -72,7 +72,20 @@ func (w *Worker) runAutoscaler(ctx context.Context) error {
...
+
+		w.logger.Info("succeeded in autoscaling:", logInfo...)
```
Comment (begelundmuller) [inline admin/worker/run_autoscaler.go:88]: nit: remove the trailing `:` in the message – this is structured logging, so formatting is already taken care of

**Review by begelundmuller — CHANGES_REQUESTED** [review]

File: admin/worker/run_autoscaler.go (hunk 1 above)
Comment (Kavinjsir) [reply admin/worker/run_autoscaler.go]: Sounds good!

File: admin/worker/run_autoscaler.go (hunk 2 above)
Comment (Kavinjsir) [reply admin/worker/run_autoscaler.go]: Thx for letting me know 😃  Updated other places also correspondingly

**Comment by Kavinjsir** [comment]
@begelundmuller Thx for reviewing with suggestions! Just updated code, can you help with another review :) 🏓

**Review by begelundmuller — APPROVED** [review]

**Pull request merged**